lms/djangoapps/courseware/features/lti.py

```diff
@@ -16,6 +16,8 @@
 
 from common import visit_scenario_item
 
+TEST_COURSE_NAME = "test_course_a"
```

Comment:
So was part of the issue that when moving this to another shard theres another test called 'test_course' from a different test or something?